newton/_src/solvers/kamino/_src/linalg/factorize/rcm_batch.py

```
@@ -331,17 +333,22 @@ def fused_rcm_tile_kernel(
         mio: wp.array[wp.int32],
         vio: wp.array[wp.int32],
         perm: wp.array[wp.int32],
+        permutation_valid: wp.array[wp.int32],
+        permutation_dim: wp.array[wp.int32],
     ):
         b, lane = wp.tid()
         if b >= num_blocks:
             return
-
         n_b = dims[b]
+        if reuse_permutation and permutation_valid[b] != int(0) and permutation_dim[b] == n_b:
```

Ruff (0.15.21)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)